src/AbstractStore.php

```diff
@@ -63,9 +63,13 @@ public function fetch($url)
 
         $ttl = 0;
         if ($res->hasHeader('Cache-Control')) {
+            $cacheControl = $res->getHeader('Cache-Control');
+            if (is_array($cacheControl)) {
```

Comment:
Oh wow, apparently this should *always* be an array. Guess this never happened because we only recently had the broker send it!

Can you instead use just `getHeaderLine`? Appears to be valid for `Cache-Control`, as per: http://stackoverflow.com/a/5240104/224129